packages/backend/src/extension.ts

```diff
@@ -45,6 +45,10 @@ export async function activate(extensionContext: ExtensionContext): Promise<void
     );
   }
 
+  telemetryLogger.logUsage('start', {
+    version: version,
```

Comment:
AFAIK if you do start, podman desktop will enrich the data with the extension version automatically

Reply:
I mean, podman desktop add always extra properties including the version

Reply:
Ah, I was looking at all the other properties and didn't notice it was adding version there: properties/common.extensionVersion. Confirmed and PR changed.

I noticed load/activate extension events from PD only include the extension id and not version, might be a better long-term practice if we just added it there.